File: ms-story/src/main/kotlin/com/example/msstory/dto/StoryDto.kt
package com.example.msstory.dto

import java.util.Date

// DTO para la entidad Story
data class StoryDto(
        val storyId: Long,
        var writerId: Long,
        val categoryId: Long,
        val title: String,
        val urlCover: String,
        val description: String,
        val audience: String,
        val language: String,
        val publicationDate: Date,
        val votes: Long,
        val status: Boolean
)

File: frontend-storyverse/src/app/dto/story.dto.ts
export interface StoryDto {
    storyId: number;
    writerId: number;
    categoryId: number;
    title: string;
    urlCover: string;
    description: string;
    audience: string;
    language: string;
    publicationDate: Date;
    votes: number;
    status: boolean;
}